src/script/conversation/ConversationVerificationStateHandler/MLS/MLSStateHandler.ts

```diff
@@ -120,6 +122,64 @@ class MLSConversationVerificationStateHandler {
     await this.checkAllConversationsVerificationState();
   };
 
+  private checkUserHandle = (identity: WireIdentity, user: User): boolean => {
+    // WireIdentity handle format is "{scheme}%40{username}@{domain}"
+    // Example: wireapp://%40hans.wurst@elna.wire.link
+    const {handle: identityHandle} = identity;
+    // We only want to check the username part of the handle
+    const regex = /.*%40([^@]+)@[^@]+/;
```

Comment:
This looks wrong having to hardcode that. I don't think we should make any assumption to what this format is going to be sadly